decidim-assemblies/app/permissions/decidim/assemblies/permissions.rb

```diff
@@ -172,6 +173,20 @@ def user_can_read_assembly_list?
         toggle_allow(user.admin? || has_manageable_assemblies?)
       end
 
+      # The list of assemblies the user is able to list
```

Comment:
I'd change this to something like:

```ruby
# Checks whether the user can list the current given assembly or not.
# 
# In case of user being admin of child assembly even parent assembly
# should be listed to be able to navigate to child assembly
```